### ISSUE TITLE
Hax issues with WAVE

### ISSUE THREAD

**Issue by JAlbert2** [elmsln/issues#535]
The WAVE tool, when it worked with Hax websites, alerted to only one issue; there are no page regions. This may just be because I made a super basic webpage with little detail, but the IST 402 course has the same issue. Simple solution to this though. WAVE suggests using more header, nav, main, and similar tags in the site creators. 

Side note: WAVE also cannot load or recognize the main Hax page. Unsure as to why.

**Comment by btopro** [comment]
Thank you for the report @JAlbert2 . @collinkleest this could be a good step into the HAXcms theme layer as that's what is delivering the elements in question. If we used header, main, footer, etc it would be nicer for wave but also certain devices that allow for "reader mode"

**Comment by btopro** [comment]
related to #536 I did this via screen share for the clean-one; other themes to audit and apply as necessary
- [ ] outline-player
- [ ] clean-two
- [ ] haxor-slevin
- [ ] simple-blog
- [ ] haxcms-slide-theme
- [ ] haxcms-blank-theme
- [ ] learn-two-theme

**Comment by btopro** [comment]
`window.HAXCMS.instance.store.manifest.metadata.theme.element` hack to modify the store directly

**Comment by btopro** [comment]
PR fixed as well.
